articles/quickstart/spa/_includes/_getting_started.md

```diff
@@ -1,13 +1,16 @@
-This integration guide will walk you through setting up and managing authentication and authorization in your ${library} apps using Auth0.
+This guide walks you through setting up authentication and authorization in your ${library} apps with Auth0.
```

Comment:
As discussed on slack, this is a placeholder that is replaced when the snippet is inserted into an article (the variable is defined in the insert command).